runtime/common/src/parachains.rs

```diff
@@ -618,7 +618,7 @@ impl<T: Trait> Module<T> {
 	/// invoked off-chain.
 	///
 	/// Yields a structure containing all unrouted ingress to the parachain.
-	pub fn ingress(to: ParaId, since: Option<T::BlockNumber>) -> Option<Vec<(T::BlockNumber, BlockIngressRoots)>> {
+	pub fn ingress(to: ParaId, since: Option<T::BlockNumber>) -> Option<Vec<(T::BlockNumber, Vec<(ParaId, Hash)>)>> {
```

Comment:
you can just remove this function

Reply:
(watermarks, `UnroutedIngress` storage and logic using them can be removed as well)

Reply:
along with the relevant `ParachainHost` functions